src/components/views/rooms/RoomHeader.tsx

```diff
@@ -215,6 +216,8 @@ export default function RoomHeader({
                     </Tooltip>
                 )}
 
+                <VideoRoomChatButton room={room} />
```

Comment:
Might be worth having a comment to say it renders nothing outside of video rooms to avoid confusion without jumping into its source